source/docs/software/driverstation/manually-setting-the-driver-station-to-start-dashboard.rst

```diff
@@ -0,0 +1,47 @@
+Manually Setting the Driver Station to Start Dashboard
+======================================================
+
+.. note:: If WPILib is not installed to the default location (such as when files are copied to a PC manually), the "C++" or "Java" buttons may not properly launch the dashboard of choice. To have the DS start a dashboard when it starts up, you have to manually modify the settings for the default dashboard.
+
+.. warning:: This is not needed for most installations, try using the appropriate :ref:`Dashboard Type setting <docs/software/driverstation/driver-station:Setup Tab>` for your language first.
+
+Set Driver Station to Default
+-----------------------------
+
+.. image:: images/manually-setting-the-driver-station-to-start-smartdashboard/set-ds-to-default.png
+
+Open the Driver Station software, click on the Setup tab and set the Dashboard setting to Default. **Then close the Driver Station!**
+
+Locate Dashboard JAR file
+-------------------------
+
+Find the location of the dashboard jar file (SmartDashboard or Shuffleboard). It should be installed to C:\Users\USERNAME\wpilib\tools where USERNAME is the username of the user logged into the computer.
```

Comment:
Need to escape the path

Reply:
Or better, replace it with an inline formatting block ``C:\Users\Public\Documents\FRC``